### DISCUSSION TITLE
Pandoc filter which populates metadata from marked definition lists

### DISCUSSION THREAD

**Discussion by bpj** [jgm/djot#327] [Show and tell]
>I like [djot] a lot, and it is now supported by Pandoc, but it has no native format for in-document metadata which is a major hurdle for me. This filter is meant to remove that hurdle by “converting” specially marked djot definition lists into Pandoc metadata.

https://github.com/bpj/pandoc-list2meta

This is basically my idea for how metadata in djot should work, so I encourage others who are or think about using djot with Pandoc to try it and share their experiences. I obviously think that it works quite well after the relatively short time I have been trying it out.

**Comment by uvtc** [comment]
Thanks, @bpj . Can you provide a toy example here of what a typical doc header might look like (with author, tags, date, etc.)?

**Reply by bpj** [reply]
There is an example in the README in the linked repo.
